source/events/issue_handler/actions/created/commands/go/source/breaking-changes/findBreakingChanges.ts

```diff
@@ -62,6 +63,11 @@ Avoid explanations and reply with a JSON following this schema:
   "required": ["breaking_changes"]
 }`;
 
+const breakingChangesDoubleCheckPrompt = `Please review your analysis, paying close attention to the following:
+
+- For each breaking change found, double check that is fits the provided definition of "breaking change". If it doesn't, remove it from the list.
```

Comment:
What I write below comes from my personal experience using Chat GPT 4 on the website:

The more messages and the longer messages I write, the less it remembers. Thus, given that prompt + changelog can be many tokens, I would paste "breaking changes" together with the double check prompt so that they are available to the model right there.

Reply:
Furthermore, I would not write "Please review your analysis," but rather treat it as a separate prompt - here is a list of changes that were flagged as breaking changes. Breakign change is "..." and you have to double check the changes and determine if they indeed are breaking.

I am just concerned that prompting it to "Please review your analysis," given a long prompt messes with the output because it's long term memory is already full. When model's memory is full my chat gpt chats act weird and i get much better results simply opening a new window and asking a fresh question.